app/ledger.js

```diff
@@ -287,18 +286,6 @@ if (ipc) {
     if (balanceTimeoutId) clearTimeout(balanceTimeoutId)
     balanceTimeoutId = setTimeout(getBalance, 5 * msecs.second)
   })
-
-  ipc.on(messages.OPEN_LEDGER_TRANSACTION_CSV, (event, viewingIds, csvFilename) => {
```

Comment:
please also remove `messages.OPEN_LEDGER_TRANSACTION_CSV` and `aboutActions.receiptLinkClick`